Misc/NEWS.d/next/Library/2024-08-23-22-01-30.gh-issue-76960.vsANPu.rst

```diff
@@ -0,0 +1,4 @@
+Fix :func:`urllib.parse.urljoin` and :func:`urllib.parse.urldefrag` for URIs
+containing empty components. For example, :func:`!urljoin()` with relative
+reference "?" now sets empty query and removes fragment. Empty components are
+preserved in the result.
```

Comment:
What is meant by " Empty components are preserved in the result." in this news entry?

Reply:
Got it what is meant in the ticket summary. 

```
Preserve empty components (authority, params, query, fragment) in urljoin().
Preserve empty components (authority, params, query) in urldefrag().
```

Reply:
I guess that I should be more explicit (even if a little repetitive) here.